app-directory/redirect-with-fallback/next.config.js

```diff
@@ -0,0 +1,11 @@
+const redirects = require('./redirects').complexRedirects
+
+/** @type {import('next').NextConfig} */
+module.exports = {
+  experimental: {
+    appDir: true,
+  },
```

Comment:
This is not needed anymore.
```suggestion
```

Reply:
good catch, i've updated next to the latest stable version and removed this